### PR TITLE
[WebSocket] Add configuration metadata store url setting in websocket

### PR DESCRIPTION
**Pull request by RobertIndie**
Master Issue: #13760 

### Motivation

As per #13077, `configurationStoreServers` need to be renamed to `configurationMetadataStoreUrl`.

### Modifications

* Add `configurationMetadataStoreUrl` setting to websocket
* Deprecate `configurationStoreServers` in websocket

### Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)
  - The schema: (no)
  - The default values of configurations: (no)
  - The wire protocol: (no)
  - The rest endpoints: (no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no)

### Documentation

- [x] `doc-required` 
  
#13758 contains doc changes for this PR.

### CODE REVIEW

File: pulsar-websocket/src/main/java/org/apache/pulsar/websocket/service/WebSocketProxyConfiguration.java
Comment (codelipenghui) [inline pulsar-websocket/src/main/java/org/apache/pulsar/websocket/service/WebSocketProxyConfiguration.java:75]: Please help also update the `websocket.conf`, following the same pattern of `broker.conf` to move the deprecated configurations to the end of the config file.

**Comment by Anonymitaet** [comment]
Hi @RobertIndie I've removed `doc-required` and added `doc-added` since the doc has been added.

BTW, https://github.com/apache/pulsar/pull/13758 can be merged after this PR gets merged?

**Comment by RobertIndie** [comment]
> Hi @RobertIndie I've removed `doc-required` and added `doc-added` since the doc has been added. Please pay attention to this next time, thanks!

Thanks.

> BTW, #13758 can be merged after this PR gets merged?

Yes. #13758 already contains the doc for this PR.

File: pulsar-websocket/src/main/java/org/apache/pulsar/websocket/service/WebSocketProxyConfiguration.java
Comment (nodece) [inline pulsar-websocket/src/main/java/org/apache/pulsar/websocket/service/WebSocketProxyConfiguration.java:75]: Looks like we should be compatible with the old config, overwrite the `getConfigurationMetadataStoreUrl()`, so like:
```java
    public String getConfigurationMetadataStoreUrl() {
        if (StringUtils.isNotEmpty(this.configurationMetadataStoreUrl)) {
            return this.configurationMetadataStoreUrl;
        }

        // backward compatible
        if (StringUtils.isNotEmpty(this.configurationStoreServers)) {
            return this.configurationStoreServers;
        }
        return this.globalZookeeperServers;
    }
```

Comment (RobertIndie) [reply pulsar-websocket/src/main/java/org/apache/pulsar/websocket/service/WebSocketProxyConfiguration.java:75]: This has already been done in `ServiceConfiguration.getConfigurationMetadataStoreUrl`. When initializing the `WebSocketService`, we will convert `WebSocketProxyConfiguration` to `ServiceConfiguration` via `PulsarConfigurationLoader.convertFrom`. Backward compatibility has been done in `ServiceConfiguration`. 

> You also need to check the usage, use `WebSocketProxyConfiguration#getConfigurationMetadataStoreUrl()` instead of `globalZookeeperServers()` and `getConfigurationStoreServers()`.

I have already checked that. There are no direct use of `WebSocketProxyConfiguration#getConfigurationMetadataStoreUrl()` and there shouldn't be.

**Review by codelipenghui — APPROVED** [review]

**Review by nodece — APPROVED** [review]
LGTM

**Pull request merged**
